blazor/datagrid/columns.md

```diff
@@ -643,6 +643,107 @@ The Header Template has options to display custom element value or content in th
 The following screenshot represents the Header Template.
 ![Blazor DataGrid with Header Template](./images/blazor-datagrid-header-template.png)
 
+## Change the orientation of header text
+
+You can change the orientation of the header text by using the [CustomAttributes](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_CustomAttributes) property of the [GridColumn](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html).
+
+Ensure the following steps to rotate the header text of particular column.
+
+**Step1:**
+
+Add the custom CSS class to a particular column by using the `CustomAttributes` property of the `GridColumn`.
+
+```cshtml
+    <GridColumn Field=@nameof(Order.CustomerID) HeaderText="Customer Name" TextAlign="TextAlign.Center" CustomAttributes="@(new Dictionary<string, object>(){ { "class", "customerclass" }})" Width="150">
+    </GridColumn>
+```
+
+**Step2:**
+
+Create a CSS class with orientation style for the grid header cell.
+
+```cshtml
+    .e-grid .e-columnheader .e-headercell.orientationclass .e-headercelldiv { // Rotate a particular headertext
+        transform: rotate(45deg);
+    }
+```
+
+**Step3:**
+
+Resize the header cell height by using the following code.
+
+```cshtml
+function setHeaderHeight(args) {
+    var textWidth = document.querySelector(".e-headercell > div").scrollWidth; // obtain the width of the headerText content.
```

Comment:
code is wrong here it is not needed to set height for all headercell instead set the height to tr.e-columnheader